ckanext/ioos_theme/lib/mailer.py

```diff
@@ -0,0 +1,580 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+'''
+catalog_harvesting/notify.py
```

Comment:
This doesn't match the filename: ckanext/ioos_theme/lib/mailer.py

Is this where the package originally came from?